ui/ui.go

```diff
@@ -225,6 +225,12 @@ type dest struct {
 func (o *Object) init() {
 	var p interface{}
 	switch o.Kind {
+	case "bandLimitedSaw":
```

Comment:
If this is strictly better than the existing oscillators then I'd rather replace them, and use the simpler names "saw", "square", and "triangle".